core/lib/libmbedtls/src/sub.mk

```diff
@@ -1 +1,16 @@
 srcs-y += tee_lmd_provider.c
+
+# For mbedtls
+cflags-y += -Icore/lib/libmbedtls/src/mbedtls/include
+cflags-y += -D'MBEDTLS_CONFIG_FILE="mbedtls_config.h"'
```

Comment:
Please put -D stuff in cppflags-y instead.

Reply:
Sorry for not quite understand it. Could you please explain why use cppflag for "-D"?

Reply:
-D is a flag is belongs to the C preprocessor which also is used for .S files. Not that it matters much here, but it's good to be consistent.

Reply:
Ok, got.